treeherder/webapp/api/performance_serializers.py

```diff
@@ -458,6 +458,9 @@ class PerfCompareResultsSerializer(serializers.ModelSerializer):
     delta_percentage = PerfCompareDecimalField()
     magnitude = PerfCompareDecimalField()
     new_is_better = OptionalBooleanField()
+    is_confident = OptionalBooleanField()
+    more_runs_are_needed = OptionalBooleanField()
+    noise_metric = OptionalBooleanField(default=False)
```

Comment:
perhaps I'm missing something, but I don't see that `noise_metric` is being calculated anywhere. Did you intend to do this in a different patch?

Reply:
Yes, I'll add it in another PR, the logic is not straightforward for this and I needs to investigate the code some more, by default is false, and this is what I set here.

Reply:
gotcha :+1: